src/console_cmds.cpp

```diff
@@ -988,7 +988,7 @@ DEF_CONSOLE_CMD(ConExec)
 	}
 
 	if (ferror(script_file)) {
-		IConsoleError("Encountered error while trying to read from script file");
+		IConsolePrint(CC_ERROR, "Encountered error while trying to read from script file %s", argv[1]);
```

Comment:
Similar here, and with others. Might help for weird cases, like non-breaking-white-space etc :P